doc/API.md

```diff
@@ -196,6 +196,47 @@ height
 
 ```
 
+###### Response
+The JSON formatted block or a standard error response.
+```
+{
+    "minerpayouts": [
+        {
+            "unlockhash": "68a0607b15f21ce643bb47075e539b9e938d7a8e0870951a3c872a2d89ae05194ee3a0f18680",
+            "value": "299999000000000000000000000000"
+        }
+    ],
+    "nonce": [
+        253,
+        253,
+        0,
+        0,
+        0,
+        0,
+        0,
+        100
+    ],
+    "parentid": "25f6e3b9295a61f69fcb956aca9f0076234ecf2e02d399db5448b6e22f26e81c",
+    "timestamp": 1433626546,
+    "transactions": [
+        {
+            "arbitrarydata": [
+                "Tm9uU2lhQ9ikb2Lc6jDRfLhhjPflTQ=="
+            ],
+            "filecontractrevisions": [],
+            "filecontracts": [],
+            "minerfees": [],
+            "siacoininputs": [],
+            "siacoinoutputs": [],
+            "siafundinputs": [],
+            "siafundoutputs": [],
+            "storageproofs": [],
+            "transactionsignatures": []
```

Comment:
I think these will actually be `null`, not `[]`. Also, if we're going to provide a real example, might be best to use one with inputs+outputs instead of just arbitrary data.

Reply:
This is actually a real example using `height=1` formatted using the python json tool. 
Not sure if having a bigger block is a good idea since even a small block can easily have 300 lines of json.